.changeset/olive-elephants-protect.md

```diff
@@ -0,0 +1,17 @@
+---
+"@comet/admin": major
+---
+
+Remove `DialogContent` from `EditDialog` as spacing inside a dialog is not commonly needed in the Comet DXP design
```

Comment:
It is actually quite common that we need the spacing, it's just also common to not need it 😄 

```suggestion
Remove `DialogContent` from `EditDialog` as spacing inside a dialog is not always needed in the Comet DXP design
```